### ISSUE TITLE
SurfaceProperty:ConvectionCoefficients not working properly

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#10523]
Issue overview
--------------
User file has these convection inputs:
```
SurfaceConvectionAlgorithm:Inside,
    CeilingDiffuser;         !- Algorithm
SurfaceConvectionAlgorithm:Outside,
    SimpleCombined;          !- Algorithm
SurfaceProperty:ConvectionCoefficients,
    Ceiling Path 1N,         !- Surface Name
    Inside,                  !- Convection Coefficient 1 Location
    TARP,                    !- Convection Coefficient 1 Type
    ,                        !- Convection Coefficient 1 {W/m2-K}
    ,                        !- Convection Coefficient 1 Schedule Name
    ,                        !- Convection Coefficient 1 User Curve Name
    ,                        !- Convection Coefficient 2 Location
    ,                        !- Convection Coefficient 2 Type
    0.1;                     !- Convection Coefficient 2 {W/m2-K}

and the same for 
    Ceiling Path 1S,         !- Surface Name
    Ceiling Path 2,          !- Surface Name
```

The err file has these warnings, and the ceiling surfaces are using CeilingDiffuser, not TARP.
```
   ** Warning ** User Supplied Convection Coefficients, Multiple Surface Assignments="CEILING PATH 1N", there were no surfaces of that type found for INSIDE assignment.
   ** Warning ** User Supplied Convection Coefficients, Multiple Surface Assignments="CEILING PATH 1S", there were no surfaces of that type found for INSIDE assignment.
   ** Warning ** User Supplied Convection Coefficients, Multiple Surface Assignments="CEILING PATH 2", there were no surfaces of that type found for INSIDE assignment.

```

### Workaround
Use this instead:
```
  SurfaceProperty:ConvectionCoefficients:MultipleSurface,
    AllExteriorRoofs,      !- Surface Type
    Inside,                 !- Convection Coefficient 1 Location
    TARP;                   !- Convection Coefficient 1 Type
```

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus 24.2.0-77d36414bc

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added EnergyPlusDevSuppor\DefectFiles...
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by jmarrec** [comment]
Gone in 23.2.0 though

![image](https://github.com/NREL/EnergyPlus/assets/5479063/718f736a-f465-41c7-aef5-98dcdf23ca69)

**Comment by RKStrand** [comment]
Just tried this in current develop (Version 24.2.0-7b75d61342) and it is no longer producing this error message.

@mjwitte Did it somehow get fixed between V23.1 and now?

**Comment by mjwitte** [comment]
I just retested things. Apparently it's been fixed at some point. Sorry for the confusion. Closing this.
